backend/src/libs/packages/server-application/libs/helpers/create-error-response/handlers/handle-auth-error-response.ts

```diff
@@ -0,0 +1,28 @@
+import { ServerErrorType } from '#libs/enums/enums.js';
+import { AuthError } from '#libs/exceptions/exceptions.js';
+import { HTTPCode } from '#libs/packages/http/http.js';
+
+import { errorInfoLabelMap } from '../../error-info-label-map/error-info-label-map.js';
+import { type ErrorHandler } from '../types/types.js';
+
+const handleAuthErrorResponse: ErrorHandler = (error) => {
+  if (error instanceof AuthError) {
+    const { message } = error;
+    const status = HTTPCode.UNAUTHORIZED;
+
+    return {
+      status,
+      info: `${
+        errorInfoLabelMap[error.constructor.name]
```

Comment:
Let't remove this mapper, we will use this value only once and won't iterate over it